src/weather_magic/core.cljs

```diff
@@ -67,22 +85,37 @@
              (assoc-in [:uniforms :uvLeftRightOffset] (if (= left-right-key :left) (* 0 1.0) (* 0.5 1.0)))
              (assoc-in [:uniforms :uvOffset]   @state/space-offset))))
       (gl/bind (:texture base-texture) 0)
+      ;; If the data from thor has been loaded, use that instead of trump.
```

Comment:
Vi vill ha någonting vi kan visa om Thor är nere, om vi bara har ingenting så kraschar programmet. Men det kan vara en god ide att byta ut bilden mot någon jätteliten bild som säger "Failed to load data". Skapar ett ärende på det #96.